openstates/importers/base.py

```diff
@@ -298,9 +298,6 @@ def import_data(
         for json_id, data in self._prepare_imports(data_items):
             obj_id, what = self.import_item(data)
             if not obj_id or not what:
-                self.logger.warning(
-                    f"Skipped {data} because it did not have an associated ID or type"
```

Comment:
Is there a shortened version of this message we could keep? Hate to lose logging that was considered useful at one point.

Reply:
Makes sense, will add a message there